install/linux/usr/share/odemis/sim/sparc2-ek-sim.odm.yaml

```diff
@@ -2,13 +2,6 @@
 SPARC2: {
     class: Microscope,
     role: sparc2,
-    children: ["SEM E-beam", "SEM Detector", "Calibration Light",
```

Comment:
Officially: because the old definition should still work. So it's good to test with both ways.
Unofficially: because I couldn't be bothered updating 30+ files ;-)